### PR TITLE
Question

### PR DESCRIPTION
**Pull request by romabulani**
- Added Question Page
- Made some style changes in shared files

[Here](https://deploy-preview-5--bakinmania.netlify.app/) is the preview link 

question

### CODE REVIEW

File: src/components/question/question.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  margin: 0.5rem;
+}
+
+.options {
+  width: 100%;
+}
+
+.option {
+  appearance: none;
+}
```
Comment (pranjalmahajan96) [inline src/components/question/question.css]: name your classes properly

File: src/components/question/question.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            <button class="btn btn-primary">Next >></button>
+          </div>
+        </div>
+      </main>
+      <!-- MIDDLE SECTION ENDS-->
+
+      <!-- FOOTER -->
+      <footer class="footer-container flex-column-center">
+        <p class="footer-header">
+          Made with <span>&lt;/&gt;</span> by Roma Bulani
```
Comment (pranjalmahajan96) [inline src/components/question/question.html:112]: we can look for a similar way as template literals to integrate symbols

File: src/shared/reset.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-    padding: 0%;
-    font-size: 16px;
-  }
-  
\ No newline at end of file
+  box-sizing: border-box;
+  font-size: 16px;
+  margin: 0%;
+  padding: 0%;
+}
```
Comment (pranjalmahajan96) [inline src/shared/reset.css:6]: add a new line at the end to get rid of this error

**Review by vinc3nati — APPROVED** [review]
Looks good!

File: src/components/rules/rules.html [hunk 4]
```diff
@@ -90,7 +90,12 @@ <h3 class="text-center heading3">Cakes Quiz</h3>
               to previous one.
             </li>
           </ul>
-          <button class="btn btn-primary">Start Quiz</button>
+          <button
+            class="btn btn-primary"
+            onclick="location.href = '../question/question.html'"
```
Comment (manish591) [inline src/components/rules/rules.html:95]: I don't know about this. Learned a new thing.

**Review by manish591 — COMMENTED** [review]
Code is looking good.

**Review by manish591 — APPROVED** [review]

File: src/components/question/question.html (hunk 2 above)
Comment (romabulani) [reply src/components/question/question.html:112]: I looked for multiple blogs and resources, they have done using template literals in jsx, will change these when we will add jsx in React

**Pull request merged**